docs/static/hour-of-code/educators.html

```diff
@@ -0,0 +1,168 @@
+<!DOCTYPE html>
+<html
+    lang="en-US"
+    xmlns="http://www.w3.org/1999/xhtml"
+    xmlns:og="http://ogp.me/ns#"
+    xmlns:fb="http://www.facebook.com/2008/fbml"
+>
+    <head>
+        <meta charset="UTF-8" />
+        <title>Microsoft MakeCode Arcade Hour of Code 2020</title>
+        <meta
+            name="Description"
+            content="Microsoft MakeCode Arcade Hour of Code 2020"
+        />
+        <meta
+            name="viewport"
+            content="width=device-width, initial-scale=1.0, user-scalable=yes"
+        />
+
+        <!-- @include scripthead.html -->
+        <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,200;0,300;0,400;0,500;0,600;1,200;1,300;1,400;1,500;1,600&display=swap" rel="stylesheet">
+        <link rel="stylesheet" href="./styles.css" />
+
+        <script>
+            function handleOnLoad() {
+                document
+                    .querySelector('#openhome')
+                    .addEventListener('click', function () {
+                        window.pxtTickEvent('hour of code.home');
```

Comment:
Can't have spaces in the ticks. Just say hourofcode.home

Reply:
Better hourofcode2020.home

Reply:
got it!